code/game/objects/effects/decals/cleanable.dm

```diff
@@ -17,11 +17,66 @@
 
 	var/on_wall = 0 //Wall on which this decal is placed on
 
-/obj/effect/decal/cleanable/New()
-	if(random_icon_states && length(src.random_icon_states) > 0)
+	var/persistence_type = SS_CLEANABLE
+	var/age = 1 //For map persistence. +1 per round that this item has survived. After a certain amount, it will not carry on to the next round anymore.
+	var/persistent_type_replacement //If defined, the persistent item generated from this will be of this type rather than our own.
+	/obj/item/trash/New(var/loc, var/age, var/icon_state, var/color, var/dir, var/pixel_x, var/pixel_y)
```

Comment:
the indentation is wrong

Reply:
oh actually it's a duplicate definition, trash.dm:14

Reply:
How did this even compile?

Reply:
http://www.byond.com/forum/?post=2408847

It's a feature apparently